website/blog/2019-05-19-15-verdaccio-4-release.md

```diff
@@ -0,0 +1,219 @@
+
+You can install and upgrade to the latest version by following commands:
+using **npm** 
+````
+npm install -g verdaccio@4.0.0
+````
+
+or using **Yarn** 
+````
+yarn global add verdaccio@4.0.0
+````
+
+or using **pnpm** 
+````
+yarn global add verdaccio@4.0.0
+````
+
+You can find detailed installation instructions [here](https://verdaccio.org/docs/en/installation)
+
+## Why 'Freedom' ?
+Verdaccio originated from [sinopia](https://github.com/rlidwka/sinopia) almost three years ago and since then verdaccio team maintaining and releasing major release every year. Since the fork, the project has evolved in many ways, making the project’s code base modern, easier to debug and more straightforward to contribute to by the community. 
+
+The name freedom holds true meaning for version 4 release. Verdaccio is a strong community of many contributors and developers from across the world, providing an ideal platform for everyone to give control of their code. Also, version 4 is free from tech debt of legacy code and stands on design patterns of the modern era which consist React, Typed plugin system, JWT, Docker & kubernetes. We can call it Freedom in true sense.
+
+Let's take a quick look at the life cycle and development of verdaccio community:
+
+- **Verdaccio@2 (Release name: Birth)** -  focused on stability, code quality, improvement in  architecture of the old sinopia project and the most important community development. 
+
+- **Verdaccio@3 (Release name: Hope)** - revamped the user interface in React and introduced the simplicity of the plugin development. The verdaccio team and many contributors made the project almost bug free and robust. This was the time project started to grow and big players started using it. 
+
+It's the first release, verdaccio is coming up with many exciting new CLI commands for package management,  Fast and Responsive user interface,  Security upgrades and easy deployments.
+
+Excited ?? Yes !!! Let's go !!
+
+## So what's changed ? TL;DR
+- [New User Interface](#new-ui)
+    - [New search Process](#new-search-process)
+    - [Packages](#package-card)
+    - [Registry information](#register-info)
+    - [Detailed Page](#detailed-page)
+    - [Package sidebar](#package-sidebar)
+- [New Router APIs](#router-api-ui)
+- [Improvements in package access](#package-access)
+- [Disable Gravatar](#disable-gravatar)
+- [New commands](#new-npm-commands)
+    - [npm star](#npm-star)
+    - [npm profile](#npm-profile)
+- [Plugins](#plugins)
+- [Drop node 6 / npm 3 support](#remove-node-6)
+- [Unpublish packages role](#unpublish)
+- [JWT token](#jwt-token)
+- [Docker improvements](#docker-improvements)
+- [Tech updates](#tech-updates)
+    - [eslint config](#verdaccio-eslint-config)
+    - [Verdaccio babel preset](#verdaccio-babel-preset)
+    - [UI Plugin](#vedaccio-ui-plugin)
+    - [Github actions](#github-actions)
+    - [Meetup & conferences](#meetup-conferences)
+
+## <a id="new-ui"></a> New user interface
```

Comment:
I didn't remember if this is a standard of Markdown, but when setting a heading (\##) it creates a link automatically (in this case it should be new-user-interface). Do we need to create anchors??